src/main/java/com/github/couchmove/service/ChangeLockService.java

```diff
@@ -31,11 +32,11 @@ public class ChangeLockService {
     private String uuid;
 
     public ChangeLockService(Bucket bucket, Cluster cluster) {
-        this.repository = new CouchbaseRepositoryImpl<>(cluster, bucket, ChangeLock.class);
+        this.repository = new CouchbaseRepositoryImpl<>(cluster, bucket, ChangeLock.class, Collections.emptyMap());
```

Comment:
This change can be avoided by adding new constructor without variables

Reply:
Done